### PR TITLE
make PROCESS_INPUT optional

### PR DESCRIPTION
**Pull request by KateSakharova**
PROCESS_INPUT step renames files and changes ERZ to ERR in fasta files. If we have input not from ENA - that step should be skipped.

### CODE REVIEW

File: workflows/genomes_generation.nf [hunk 1]
```diff
@@ -115,15 +115,23 @@ workflow GGP {
     FASTQC_BEFORE (assembly_and_runs.map{ meta, _ , reads -> [meta, reads] })
     ch_versions = ch_versions.mix(FASTQC_BEFORE.out.versions)
 
-    // ---- pre-processing ---- //
-    PROCESS_INPUT( assembly_and_runs ) // output: [ meta, assembly, [raw_reads] ]
-
-    ch_versions = ch_versions.mix( PROCESS_INPUT.out.versions )
+    def input_assemblies_and_runs
```
Comment (mberacochea) [inline workflows/genomes_generation.nf:118]: this def is not assinged, should it be an empty channel?.

File: workflows/genomes_generation.nf [hunk 2]
```diff
@@ -115,15 +115,23 @@ workflow GGP {
...
+    if ( params.skip_preprocessing_input ) {
+        println('skipping pre-processing')
```
Comment (mberacochea) [inline workflows/genomes_generation.nf:120]: There is a `log.info "Skipping pre-processing"` we could use, it's not documented in Nextflow but I've seen it in other pipelines.

**Review by mberacochea — CHANGES_REQUESTED** [review]
:)

File: workflows/genomes_generation.nf (hunk 2 above)
Comment (KateSakharova) [reply workflows/genomes_generation.nf:119]: maybe later when we change logging for whole pipeline and remove all `print-s`. It requiers more time and testing

**Pull request merged**